NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj

```diff
@@ -23,6 +23,13 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="4.49.0" />
```

Comment:
What do these changes actually do?

Reply:
It ensures that Careers Progression outcomes can be claimed within 13 months of their 'Outcome Effective Date' (which is in line with the outcome rule/business process). Before this change, an error is raised if 12 months have passed which is incorrect.

Reply:
I commented in the csproj file - do you need these changes to be committed?
@lschonauer-DFE

Reply:
My mistake. I'm not sure to be honest. I think those changes were added automatically when I tried running this locally. They related to 'ignored' files so it should be OK.
![Screenshot 2024-08-16 103308](https://github.com/user-attachments/assets/c0ab2209-9a08-40bd-b42f-77e824461678)

Reply:
Other blog posts appear to suggest this to be the case too: 
![image](https://github.com/user-attachments/assets/b0107b08-a290-4c44-9139-a3d79640efd3)

Reply:
Will resolve @jpotter-DFE